package.json

```diff
@@ -14,8 +16,8 @@
     "google-auth-library": "^0.12.0",
     "googleapis": "^25.0.0",
     "newrelic": "^4.9.0",
-    "probot": "^5.0.0",
-    "probot-scheduler": "^1.2.0",
+    "probot": "^9.5.3",
+    "probot-scheduler": "^2.0.0-beta.1",
```

Comment:
Why are we using the beta version?

Reply:
I noticed that the previous version `1.2.0` had some issues after the upgrade, so I decided to upgrade it as well and currently, the beta version is the latest version.

Reply:
But isn't the beta version also a version for testing and can have issues - can we confirm this on their repo once - that is it safe to use the beta version?

Reply:
It's not stated that's it's unsafe, but there is an issue on the repository https://github.com/probot/scheduler/issues/35

Also, v1.2.0 of probot-scheduler doesn't work fine with probot v9

Reply:
Ok, so here is what I would suggest: it is always better to confirm in the start. So, let us either create an issue or send a mail with our query regarding why is the latest release in beta version and is it ok to use it.

Reply:
Okay.
I've created an issue https://github.com/probot/scheduler/issues/41

Reply:
Thanks!

Reply:
> It's not stated that's it's unsafe

I think `beta` keyword states it's unstable, I would also suggest you do what @ankita240796 has suggested and make sure to thoroughly test the feature we are using from this lib.

Reply:
I've followed what @ankita240796 said by creating the issue.
Also, the feature to be used by this package is not in at the moment.
https://github.com/oppia/oppiabot/pull/54 is the first feature that uses probot scheduler.
While testing, I added little functionalities to check if it works, and it did.

Anyone else is welcome to test.
I also have a PR: jameesjohn/comment-on-pr#15 where the upgrade was being tested.

Reply:
Also, probot v9 does not work with the older version of the scheduler.

Reply:
I am fine with using beta version as long as we test everything. Did you run any check with this version and check if it works (since as of now we do not have any probot scheduler based checks).